win32_event_log/datadog_checks/win32_event_log/data/conf.yaml.example

```diff
@@ -248,28 +245,22 @@ instances:
     #
     # host: <REMOTE_HOSNAME>
 
-    ## FILTERS
-    ## When running legacy_mode least one filter is required:
-    ## `log_file`, `source_name`, `type`, `event_id`, `message_filters`
-
     ## @param log_file - list of strings - optional
     ## The `log_file` filter instructs the check to only capture events
     ## that belong to one of the specified LogFiles (Application, System, Setup, Security,
     ## or application-specific LogFile).
     ##
     ## Note: This is only used when `legacy_mode` is set to `true`.
     #
-    # log_file:
-    #   - <LOG_FILE>
+    # log_file: []
 
     ## @param source_name - list of strings - optional
     ## The `source_name` filter instructs the check to only capture events
     ## that come from one of the specified SourceNames.
     ##
     ## Note: This is only used when `legacy_mode` is set to `true`.
     #
-    # source_name:
-    #   - <SOURCE_NAME>
+    # source_name: []
 
     ## @param type - list of strings - optional - default: information
```

Comment:
What does `default: information` mean considering `event_id` is a list of strings but not a string itself.